src/pages/components/CertificateInfo.qml

```diff
@@ -103,146 +105,69 @@ SilicaFlickable {
             }
         }
 
+        DetailItem {
+            //: Label for the issuer field of a TLS certificate
+            //% "Verified by"
+            label: qsTrId("sailfish_browser-la-cert_issuer")
+            value: security ? security.issuerDisplayName : ""
+            visible: value && value.length > 0
+        }
```

Comment:
To use wrap and bind in the center this component regardless of the width, I need to take the contentWidth of the text, but with this I get a binding loop.

Reply:
Can you use `implicitWidth` rather than `width`? In other words, if `implicitWidth > parent.width - label.width` then use the full width of the parent, otherwise centre the items based on `implicitWidth + label.width`?

I'm simplifying a lot here, but the `implicitWidth` shouldn't depend on the wrapping, so shouldn't create a binding loop.